### ISSUE TITLE
Using nightly Rust

### ISSUE THREAD

**Issue by sd2k** [softprops/serverless-rust#42]
Hi! This project is awesome, so easy to get up and running, so thanks for all your work!

Is it possible to override the version of Rust used to build the functions, e.g. to use a nightly version?

**Comment by softprops** [comment]
This has been on my mind but has been a lower priority because most of the lambda work I do depends on a reliable build environment. 

Nightly rust is expressly not committed to being a stable environment to build on. That said, I'm empathetic to those that also just want to experiment with features that aren't yet complete. I'll keep this issue as a reminder of that since I don't already have an issue up for rust.

**Comment by sd2k** [comment]
Fair enough. I've managed to factor some things out into separate crates now meaning nightly is no longer required, so no problem!

For anyone finding this on Google in the meantime, it's fairly simple to fork this repo and clone it to the '.serverless-plugins' directory of your project and change the Docker image/tag in index.js; I also forked/cloned your [lambda-rust](https://github.com/softprops/lambda-rust) project to use nightly in the Dockerfile. (The same process can be used for any required static libraries, e.g. libpq).
